### PR TITLE
Replace bitbucket.org/ww/goautoneg with its mirror

### PR DESCRIPTION
**Pull request by dmage**
The package bitbucket.org/ww/goautoneg is not available anymore, so it is replaced with its mirror github.com/munnerz/goautoneg.

### CODE REVIEW

**Comment by dmage** [comment]
/assign @deads2k 

Do we need BZ for this?

**Comment by deads2k** [comment]
/approve

leaving lgtm with @sttts and/or @tnozicka

File: pkg/apiserver/httprequest/httprequest.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"strings"
 
-	"bitbucket.org/ww/goautoneg"
```
Comment (sttts) [inline pkg/apiserver/httprequest/httprequest.go:8]: I would expect a replace in go.mod.

What is upstream kube doing?

Comment (dmage) [reply pkg/apiserver/httprequest/httprequest.go:8]: replace doesn't work, upstream uses github.com/munnerz/goautoneg

```
go: github.com/munnerz/goautoneg@v0.0.0-20191010083416-a7dc8b61c822 used for two different module paths (bitbucket.org/ww/goautoneg and github.com/munnerz/goautoneg)
```

Comment (dmage) [reply pkg/apiserver/httprequest/httprequest.go:8]: https://github.com/kubernetes/kubernetes/blob/2d8fbd61afe9e2bf207be88320fd9c3001885a96/go.mod#L75

**Pull request merged**